.travis.yml

```diff
@@ -603,6 +603,7 @@ addons:
 before_install:
   - export TOXENV=$(echo $TOXENV_TMPL | envsubst)
   - pip install tox-venv tox-tags
+  - export HCLOUD_TOKEN=$(./bin/get-hetzner-cloud-token)
```

Comment:
You'd also need to put `${HETZNER_CLOUD_TTS_TOKEN}` into travis config

Reply:
Already done but it did not seem to be picking it up ... :/